systers_portal/templates/base.html

```diff
@@ -17,6 +17,12 @@
   <link rel="stylesheet" href="{% static 'css/font-awesome.min.css' %}">
 
   {% block head %}{% endblock %}
+  <script src="https://code.jquery.com/jquery-3.5.1.min.js" 
+          integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" 
+          crossorigin="anonymous">
```

Comment:
Please don't introduce jquery for this, see my comment below

Reply:
If you concern with jQuery only then @SanketDG I think this project is using jQuery, see the commented code. Difference is that I use update the version to 3.5.1 from 1.11.